packages/cursorless-engine/src/processTargets/modifiers/PreferredScopeStage.ts

```diff
@@ -22,6 +22,18 @@ export class PreferredScopeStage implements ModifierStage {
   run(target: Target): Target[] {
     const { scopeType } = this.modifier;
 
+    const containingScopeStage = new ContainingScopeStage(
+      this.modifierStageFactory,
+      this.scopeHandlerFactory,
+      { type: "containingScope", scopeType },
+    );
+
+    try {
+      return containingScopeStage.run(target);
+    } catch (_ex) {
+      // Do nothing
```

Comment:
Can we clarify why it's okay to swallow this here?

Otherwise we should just remove the comment, because it's redundant to the empty catch block

Reply:
`Empty block statement.eslint[no-empty](https://eslint.org/docs/latest/rules/no-empty)`
But sure I can elaborate